hack/Dockerfile.dev

```diff
@@ -1,7 +1,7 @@
-FROM golang:1.19
+FROM golang:1.20
 
 RUN apt-get update -y
-RUN apt-get install docker.io apt-transport-https ca-certificates gnupg python -y
+RUN apt-get install docker.io apt-transport-https ca-certificates gnupg python-is-python3 -y
```

Comment:
Don't we want to make this change in develop first?

Reply:
The change was already merged in develop